main.ts

```diff
@@ -0,0 +1,43 @@
+import { PluginInstallation } from "@amplication/code-gen-types";
+import { TestConfig } from "./types";
+
+/**
+ * Calls the API to get all the plugins from the catalog
+ * @returns all the plugins from the catalog
+ */
+function getPlugins(): PluginInstallation[] {
+  return [];
+}
+
+/**
+ * Generates all possible combinations of 4 plugins to install together
+ * @returns set of plugin combinations to install together (matrix  of all possible combinations), 4 plugins on each combination
```

Comment:
I believe that in reality, we will set the combinations manually for the most part. Or at least have some "rules" about the permutation (e.g. by plugin category).

Reply:
Of course. We must have the DB plugin, and for now, we must install auth core and auth basic/jwt together.

Reply:
I was confused by "generates all possible combinations of 4 plugins to install together" - wanted to make sure we are on the same page